pyproject.toml

```diff
@@ -71,6 +71,7 @@ dev = [
     "types-Pillow",
     "types-tqdm",
     "Flake8-pyproject",
+    "paquo"
```

Comment:
let's remove paquo from the main dependencies now that it's in the qupath extras.

```suggestion
```